plugins/auth-jwt/src/core/create-auth-module.ts

```diff
@@ -0,0 +1,71 @@
+import { Module, DsgContext } from "@amplication/code-gen-types";
+import { join } from "path";
+import { OperationCanceledException } from "typescript";
+import { templatesPath } from "../constants";
+import { readFile } from "@amplication/code-gen-utils";
+import {
+  addImports,
+  importNames,
+  interpolate,
+  removeTSClassDeclares,
+} from "../util/ast";
+import { builders, namedTypes } from "ast-types";
+import { print } from "@amplication/code-gen-utils";
+
+const authModulePath = join(templatesPath, "auth.module.template.ts");
+
+export async function createAuthModule(
+  dsgContext: DsgContext
+): Promise<Module> {
+  return await mapAuthModuleTemplate(
+    dsgContext,
+    authModulePath,
+    "auth.module.ts"
+  );
+}
+
+async function mapAuthModuleTemplate(
+  context: DsgContext,
+  templatePath: string,
+  fileName: string
+): Promise<Module> {
+  const { entities, resourceInfo, serverDirectories } = context;
+  const authEntity = entities?.find(
+    (x) => x.name === resourceInfo?.settings.authEntityName
+  );
+  if (!authEntity) throw OperationCanceledException; //todo: handle the exception
```

Comment:
Where do we catch it?

Reply:
@abrl91 Update to error to log and stop the process